tests/rational.rs

```diff
@@ -793,10 +887,9 @@ pub mod test {
 
         let guarded_rules = run_workload(terms, all_rules.clone(), Limits::rulefinding(), false);
         guarded_rules.to_file("guard.rules");
-        assert!(guarded_rules.0.contains_key("(if ?a 1 (/ 1 ?a)) ==> 1"));
```

Comment:
Wait, does this mean we don't learn this rule anymore? That's not good.

Reply:
I made the recipe simpler so it doesn't find if rules like these